go_modules/spec/fixtures/projects/invalid_pseudo_version/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/dependabot/vgotest
+
+go 1.13
+
+require (
+  rsc.io/quote v1.4.0
+  github.com/openshift/api v3.9.1-0.20190424152011-77b8897ec79a+incompatible
```

Comment:
I'm still wrapping my head around this, but it looks like go mod validate that the sha `77b8897ec79a` has the tag `v3.9.1`, which it doesn't https://github.com/openshift/api/tree/77b8897ec79a